bot.py

```diff
@@ -47,6 +48,46 @@ def does_channel_exist(channel_name):
         return False
 
 
+def prune_channels(user_token):
+    response = client.channels_list()
+    channels = response["channels"]
+    for channel in channels:
+        if channel["name"].startswith("111-change-"):
+            channel_id = channel["id"]
+            requests.get(f"https://slack.com/api/channels.delete?token={user_token}&channel={channel_id}")
+    return make_response("", 200)
```

Comment:
Why do you return a http request response from this function?

Reply:
I think the response was required to make slack not error. But yeah didn't need to be a return!